src/index.js

```diff
@@ -33,4 +33,21 @@ list.addEventListener('keypress', (e) => {
     e.preventDefault();
     Actions.editTask(e);
   }
-});
\ No newline at end of file
+});
+
+clearButton.addEventListener('click', () => {
+  Actions.isChecked();
+});
+
+list.addEventListener('change', () => { // listen for changes in chekboxes
+  const checkbox = document.querySelectorAll('.checkbox');
+  const tasks = JSON.parse(localStorage.getItem('tasks'));
+  checkbox.forEach((checkbox) => {
+    if (checkbox.checked) {
+      tasks[checkbox.id].completed = true;
+    } else {
+      tasks[checkbox.id].completed = false;
+    }
+  });
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+});
```

Comment:
- [ ] Based on the [requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m2_interactive_list.md#project-requirements), please use the filter() method to implement a function for the "Clear all completed" button

Reply:
@prolajumokeoni Hi, I'm using the filter method in the objects.js file. It gets called when the user clicks on the clear all complete button. Not sure if I understand what you mean.